### ISSUE TITLE
Some doubts about the results on shapenet55

### ISSUE THREAD

**Issue by ChenJunX1an** [junshutang/LAKe-Net#2]
I found that the results in Table1, the performance of pointr is worse than the results of PCN and GRNet, which is inconsistent with the previous papers. May I ask the author, are these results tested at the same difficulty in the PoinTr paper?
![image](https://user-images.githubusercontent.com/105265616/200536944-abe8e667-ee78-44f2-bd13-12e6e475ca3a.png)

**Comment by czvvd** [comment]
I think that's because the author partitioned the test set differently and retrained other methods.

**Comment by yuxumin** [comment]
It's strange. I don't know why the CD for PoinTr on ShapeNet-55 is worse than those in previous paper. We provide the pretrained ckpt for ShapeNet-55 in official repo.
Did the author re-train PoinTr by themselves?

**Comment by junshutang** [comment]
Sorry for the late reply. We didn't re-train PoinTr. We evaluate its pretrained model on all test set on ShapeNet-55 (no partition) and test different metrics: CD-l1/CD-l2/EMD mentioned in our paper. The evaluation code is similar to VRCNet and GRNet. We show the detailed results of these metrics in the [supplementary material](https://openaccess.thecvf.com/content/CVPR2022/supplemental/Tang_LAKe-Net_Topology-Aware_Point_CVPR_2022_supplemental.pdf).
